003-examples.md

```diff
@@ -1,36 +1,35 @@
-Examples: Mapping VO-DML ⇒ VOTable
-==================================
+# Examples: Mapping VO-DML to VOTable #
 
 This section shows examples of mappings as they may occur in realistic
 VOTables, using example Data Models. We will describe how the different
 VO-DML elements must be serialized and how annotations employing the
-&lt;VODML&gt; element can help one interpret the VOTable. In the later
+<VODML> element can help one interpret the VOTable. In the later
 normative section we turn this around and explicitly list all legal
 annotations, their constraints and interpretation.
 
 In this section we list some mappings from VO-DML to VOTable. We use
 examples extracted from a sample model with sample instances described
 in the next section. These should be seen as an introduction to the
-complete and formal specification in section 6.7.3 on how one MUST use
+complete and formal specification in section \ref{6.7.3} on how one MUST use

```

Comment:
This should be done according to the `pandoc cross-ref` syntax in order to avoid referencing numbers. Section 6.7.3 should be declared like this:

```
### Section whatever what not {#sec:whatever}
```
And the reference should be `[@sec:whatever]`.

This will not be rendered properly by the `simple.sh` script, because it doesn't require/use the `cross-ref` plugin.